app/models/group.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+#
+# Model for Housing Groups
+#
+# @attr [Integer] size The room size that the Group wants.
```

Comment:
ohno! I must have looked at an inaccurate reference when I started using it and never checked up on it